docs/superpowers/specs/2026-06-21-implant-gearset-abilities-D-pr4-design.md

```
@@ -0,0 +1,298 @@
+
+// outgoingEffects.ts
+export function outgoingAmplificationForHit(
+  attackerAbilities: Ability[],
+  ctx: OutgoingHitContext,
+  rollProc: (abilityId: string, chance: number) => boolean,
+): number   // returns the SUMMED amplification % for this one hit (0 when nothing applies)
+```
+
+Semantics (pure, deterministic given `rollProc`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)